sections/rich-text.liquid

```diff
@@ -48,6 +48,34 @@
                         "label": "Rich-text"
                     }
                 ]
+            },
+            {
+                "type": "buttons",
+                "name": "Buttons",
+                "settings": [
+                    {
+                        "type": "text",
+                        "id": "button_one_text",
```

Comment:
@sergey221997 Use convention like 'first button' and 'second button' instead of 'one', 'two'. Check Showtime for example.